fix_includes_test.py

```diff
@@ -2841,6 +2841,31 @@ def testRemoveDuplicates(self):
 #include "used.h"
 #include "used2.h"
 ---
+"""
+    self.RegisterFileContents({'remove_duplicates': infile})
+    self.ProcessAndTest(iwyu_output)
+
+  def testDontRemoveTemplateLines(self):
+    """Tests we don't accidentally think repeated template lines are dupes."""
+    infile = """\
+#include <notused.h>  ///-
+template<typename T>
+class A;
+template<typename T>
+class B;
+
+f(A&, B&);
+"""
+    iwyu_output = """\
+remove_duplicates should add these lines:
```

Comment:
Maybe name this something unique to avoid confusing it with the test before (the diff will become more obvious, too).

Also, I wonder if there's a test missing to make sure we *do* treat two identical multi-line template fwd decls as dupes?

Reply:
Good idea.  Done both.

(CI still struggling, it looks like).